lucille-plugins/lucille-api/src/test/java/endpoints/LivenessResourceTest.java

```diff
@@ -16,8 +19,24 @@ public class LivenessResourceTest {
 
   @Test
   public void testLivenessEndpoint() {
+    System.out.println("=======================================================================================================================");
     Optional<PrincipalImpl> user = Optional.of(new PrincipalImpl("test"));
-    Response response = liveness.isAlive(user);
+    Response response = liveness.isAlive();  
+    
+    // Dump relevant response details
+    System.out.println("=== Response Dump ===");
```

Comment:
please remove System.out.printlns; please use junit assertions and/or the logger

code that useful for development but not necessary for the test itself should either be removed or commented out with a quick explanation of why it's there and why someone would want to uncomment it

Reply:
removed - was trying to figure out what the intent/purpose of the two copied kubernetes like pre-existing calls were suppose to be doing.
removed